README.md

```diff
@@ -228,6 +321,9 @@ storageclass.storage.k8s.io/gold created
 <br/>
 <br/>
 
+## Upgrading
+
+In order to upgrade the CSI operator and driver from a previous version, uninstall the existing driver and then install the newer version.
```

Comment:
We do not support upgrading, we should tell the user about it, please update the words here and change it in UG accordingly

Reply:
I think it's technically upgrading, since the result is the same. it's just not seamless...

Reply:
I agree with @oriyarde  - it is still the way to upgrade the version. to say that we don't support upgrade is not a good way to word it to customers who might already be using 1.0.0.
I explain that it is to uninstall and then reinstall. that should be more than sufficient. Note that in the UG it explains how to manually upgrade (as well as how to make it automatic) via OCP.

Reply:
automatic? this specifically might be something to remove from the UG

Reply:
@oriyarde  @27149chen  please see the UG and provide any comments there, if needed.

Reply:
done